Sources/ScipioKit/Producer/Compiler.swift

```diff
@@ -22,7 +22,8 @@ struct Compiler<E: Executor> {
     }
 
     func createXCFramework(target: ResolvedTarget,
-                           outputDirectory: URL) async throws {
+                           outputDirectory: URL,
+                           force: Bool) async throws {
```

Comment:
```suggestion
                           overwrite: Bool) async throws {
```

I feel overwrite is more readable